proto/did/did.proto

```diff
@@ -0,0 +1,47 @@
+syntax = "proto3";
+package medibloc.panaceacore.did;
+
+option go_package = "github.com/medibloc/panacea-core/x/did/types";
+
+import "gogoproto/gogo.proto";
+
+message DIDDocument {
+  repeated string contexts = 1 [json_name = "@contexts"];;
+  string ID = 2 [json_name = "id"];
+  string controller = 3;
+  repeated VerificationMethod verificationMethods = 4 [json_name = "verificationMethod"];
+  repeated VerificationRelationship authentications = 5 [json_name = "authentication"];
+  repeated VerificationRelationship assertionMethods = 6 [json_name = "assertionMethod"];
+  repeated VerificationRelationship keyAgreements = 7 [json_name = "keyAgreement"];
+  repeated VerificationRelationship capabilityInvocations = 8 [json_name = "capabilityInvocation"];
+  repeated VerificationRelationship capabilityDelegations = 9 [json_name = "capabilityDelegation"];
+  repeated Service services = 10 [json_name = "service"];;
+}
+
+message VerificationMethod {
+  string ID = 1;
+  string type = 2;
+  string controller = 3;
+  string pubKeyBase58 = 4;
```

Comment:
I'm not sure whether we need to put `[json_name = "publicKeyBase58]` or not.

Reply:
Or, I guess we can use the full name `publicKeyBase58` instead of `pubKeyBase58`. The previous code wasn't clean.

Reply:
You're right!
I'll fixed it.